src/env.rs

```diff
@@ -404,6 +406,43 @@ mod tests {
         assert_eq!(reader.get("foo").expect("read"), Some(Value::I64(999)));
     }
 
+    #[test]
+    fn test_blob() {
+        let root = TempDir::new("test_round_trip_blob").expect("tempdir");
+        fs::create_dir_all(root.path()).expect("dir created");
+        let k = Rkv::new(root.path()).expect("new succeeded");
+        let sk: Store<&str> = k.create_or_open("sk").expect("opened");
+        let mut writer = sk.write(&k).expect("writer");
+
+        assert_eq!(writer.get("foo").expect("read"), None);
+        writer.put("foo", &Value::Blob(&[1, 2, 3, 4])).expect("wrote");
+        assert_eq!(writer.get("foo").expect("read"), Some(Value::Blob(&[1, 2, 3, 4])));
+
+        fn u16_to_u8(src: &[u16]) -> Vec<u8> {
```

Comment:
If this is something that users of rkv are going to have to do a lot, I'm a little confused as to why these functions are test only and not part of the API itself.

Reply:
It's a good question. XULStore needs to store UTF-16 strings with [u16] backing storage, but I'm unsure at this point how common that'll be for rkv consumers in general. Thus my inclination is to await more evidence of usage before adding an abstraction for [u16] into rkv itself.